### PR TITLE
Add SED to the glossary

### PR DESCRIPTION
**Pull request by Astro-Kirsty**
Adding SED to the gammapy glossary as per https://github.com/gammapy/gammapy/pull/5407#discussion_r1780901365

### CODE REVIEW

File: docs/user-guide/references.rst [hunk 1]
```diff
@@ -132,6 +132,10 @@ Glossary
       data are analyzed. In practice, at each energy it corresponds with the sky region
       in which the dataset mask is True.
 
+    SED
+      Short for "spectral energy distribution". For a spectral model or flux points object
```
Comment (bkhelifi) [inline docs/user-guide/references.rst:136]:
```suggestion
      Short for "spectral energy distribution". For a spectral model or flux points object, 
```

**Comment by Astro-Kirsty** [comment]
Are you happy with how this looks in the documentation?
image

**Review by bkhelifi — APPROVED** [review]
Thanks a lot

File: docs/user-guide/references.rst [hunk 2]
```diff
@@ -132,6 +132,11 @@ Glossary
       data are analyzed. In practice, at each energy it corresponds with the sky region
       in which the dataset mask is True.
 
+    SED
+      Short for "spectral energy distribution". For a spectral model or flux points
+      object, the type of plot (e.g. dN/dE, E^2 dN/dE) is typically adjusted through
```
Comment (registerrier) [inline docs/user-guide/references.rst:137]:
```suggestion
      object, the type of plot (e.g. :math:`dN/dE`, :math:`E^2\ dN/dE`) is typically adjusted through
```
using inline math type should look better.

**Review by registerrier — APPROVED** [review]
Thanks @Astro-Kirsty . Looks good see inline comment.

Note that we good point to a table listing the possible `sed_type` values. I think this the closest we have to that:
https://docs.gammapy.org/1.2/user-guide/estimators.html#flux-quantities

**Review by AtreyeeS — APPROVED** [review]
Thanks @Astro-Kirsty !

**Review by bkhelifi — APPROVED** [review]
Thanks a lot!

**Pull request merged**